### PR TITLE
feat: set pdisks column width

### PR DESCRIPTION
**Pull request by astandrik**
Closes https://github.com/ydb-platform/ydb-embedded-ui/issues/1789

[Stand](https://nda.ya.ru/t/JAevqulA7AhVjQ)

in url there are parameters for testing:
pdisks=3&vdisksPerPDisk=20&mocks=1&totalNodes=64

pdisks - maximum number of pdisks per node
vdisksPerPDisk - maximum number of vdisks per pdisk
totalNodes - total nodes in nodes list

### Test Status: <span style="color: green;">✅ PASSED</span>
  📊 [Full Report](https://ydb-platform.github.io/ydb-embedded-ui/1793/)

  | Total | Passed | Failed | Flaky | Skipped |
  |:-----:|:------:|:------:|:-----:|:-------:|
  | 232 | 232 | 0 | 0 | 0 |

  😟 No changes in tests. 😕

  ### Bundle Size: 🔺
  Current: 66.18 MB | Main: 66.17 MB
  Diff: +7.55 KB (0.01%)

  ⚠️ Bundle size increased. Please review.

  <details>
  <summary>ℹ️ CI Information</summary>

  - Test recordings for failed tests are available in the full report.
  - Bundle size is measured for the entire 'dist' directory.
  - 📊 indicates links to detailed reports.
  - 🔺 indicates increase, 🔽 decrease, and ✅ no change in bundle size.
  </details>

### CODE REVIEW

File: src/mocks/storage/nodes.ts [hunk 1]
```diff
@@ -0,0 +1,224 @@
+import {EFlag} from '../../types/api/enums';
```
Comment (astandrik) [inline src/mocks/storage/nodes.ts:1]: Will delete mocks right before merge

File: src/containers/Storage/StorageNodes/getNodes.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ import {
     NODES_COLUMNS_TO_DATA_FIELDS,
```
Comment (astandrik) [inline src/containers/Storage/StorageNodes/getNodes.ts:3]: Will delete mocks right before merge

File: src/store/reducers/storage/utils.ts [hunk 3]
```diff
@@ -243,10 +245,22 @@ export const calculateMaximumSlotsPerDisk = (
...
 
+export const calculateMaximumDisksPerNode = (
+    nodes: TNodeInfo[] | undefined,
+    providedMaximumDisksPerNode?: string,
+) => {
+    if (providedMaximumDisksPerNode) {
+        return providedMaximumDisksPerNode;
+    }
+
+    return String(Math.max(1, ...(nodes || []).map((node) => node.PDisks?.length || 0)));
```
Comment (Raubzeug) [inline src/store/reducers/storage/utils.ts:256]: it's quite hard to read. Could you please refactor?

**Review by Raubzeug — CHANGES_REQUESTED** [review]

File: src/store/reducers/storage/utils.ts (hunk 3 above)
Comment (astandrik) [reply src/store/reducers/storage/utils.ts]: mission accomplished

mind that its a fallback function for corner case of a corner case

**Review by Raubzeug — DISMISSED** [review]

**Review by Raubzeug — APPROVED** [review]

**Pull request merged**